package.json

```diff
@@ -137,12 +137,16 @@
     "test-client": "NODE_ENV=test NODE_PATH=test:client TEST_ROOT=client test/runner.js",
     "test-server": "NODE_ENV=test NODE_PATH=test:server:client TEST_ROOT=server test/runner.js",
     "test-test": "NODE_ENV=test NODE_PATH=test:client TEST_ROOT=test test/runner.js",
+    "test-client:watch": "chokidar '(client|server|test)/**/*.(js|jsx)' -c 'pkill node -f runner.js; npm run test-client ${TEST_PATH}'",
```

Comment:
@blowery,

I've looked into using both [fkill](https://github.com/sindresorhus/fkill-cli) and [cash-kill](https://github.com/dthree/cash) but each didn't fit this use case properly.  They do not allow you to specify process name & arguments the way `pkill` does.

Do you have any suggestions for how to continue here?